packages/dgrid/src/RowFormatter.ts

```diff
@@ -46,7 +46,7 @@ export class RowFormatter {
     private _columnIdx = {};
     private _formattedRow = {};
 
-    constructor(columns: ColumnType[]) {
+    constructor(columns: ColumnType[], protected _renderHtml) {
```

Comment:
This new argument isn't used

Reply:
Yes it is, when you specify a "visibility" it implies that it is a Class Data Property (and it sets it)

Reply:
Ahh okay